fraim/inputs/project.py

```diff
@@ -43,11 +44,14 @@ def __init__(self, kwargs: Any) -> None:
             self.repo_name = os.path.basename(self.project_path)
             if self.diff:
                 self.input = GitDiff(self.project_path, head=self.head, base=self.base, globs=globs, limit=limit)
+            elif kwargs.status_check:
```

Comment:
Just for consistency, 

```suggestion
            elif self.status_check:
```

and add `self.status_check = kwargs.status_check` above.